Resolve malloc and realloc declaration mismatch warnings.

```
realloc.c:27:7: warning: conflicting types for built-in function 'malloc';
expected 'void *(long unsigned int)' [-Wbuiltin-declaration-mismatch]
   27 | char *malloc ();
      |       ^~~~~~
realloc.c:26:1: note: 'malloc' is declared in header '<stdlib.h>'
   25 | #include <sys/types.h>
  +++ |+#include <stdlib.h>
   26 |
realloc.c:28:7: warning: conflicting types for built-in function 'realloc';
expected 'void *(void *, long unsigned int)' [-Wbuiltin-declaration-mismatch]
   28 | char *realloc ();
      |       ^~~~~~~
realloc.c:28:7: note: 'realloc' is declared in header '<stdlib.h>'
```

--- wol-0.7.1-orig/lib/realloc.c
+++ wol-0.7.1/lib/realloc.c
@@ -23,9 +23,7 @@
 #undef realloc
 
 #include <sys/types.h>
-
-char *malloc ();
-char *realloc ();
+#include <stdlib.h>
 
 /* Change the size of an allocated block of memory P to N bytes,
    with error checking.  If N is zero, change it to 1.  If P is NULL,
